applications/directory/src/hooks/reviewArtistProfile.js

```diff
@@ -15,17 +15,12 @@ const useReview = (token) => {
 
   const updateReview = async (values) => {
     try {
-      const updatedArtist = await httpClient.patch(
-        `/reviews/token/${token}`,
-        values
-      );
+      await httpClient.patch(`/reviews/token/${token}`, values);
       return {
-        updatedArtist
+        data: 'Artist profile reviewed successfully'
```

Comment:
No need to return `data`. The caller of `updateReview()` will know if the request was successful by using `try/catch` itself, and then the caller can formulate its own success/failure messages.

Instead, this message should return the updated artist.

```javascript
const updateReview = async (values) => {
  return await httpClient.patch(`/reviews/token/${token}`, values);
};
```

No try/catch needed inside the method.

Reply:
done